lib/cocoapods/native_target_ext.rb

```diff
@@ -0,0 +1,52 @@
+module Xcodeproj
+  class Project
+    module Object
+      class AbstractTarget
```

Comment:
is there any reason this isn't going into `Xcodeproj` ? I'm not a huge fan of re-opening another gem's classes

Reply:
Redid as a static class method in on `Project`. Let me know what you think